### ISSUE TITLE
why not add batch norm in depth decoder?

### ISSUE THREAD

**Issue by Darknesszlx** [nianticlabs/monodepth2#238]
https://github.com/nianticlabs/monodepth2/blob/101dd03f00e0d8e9bcd8a8d760f94bc2820f5549/networks/depth_decoder.py#L35
https://github.com/nianticlabs/monodepth2/blob/101dd03f00e0d8e9bcd8a8d760f94bc2820f5549/layers.py#L106-L118

**Comment by mrharicot** [comment]
Hi,

We have batchnorm in the encoder because resnet has it, however as shown in the groupnorm paper, training with both small batch sizes (< 32 which is the case for monodepth2) and batchnorm hurts accuracy.
However I don't recall adding batchnorm and comparing, it's actually possible that it could improve results.

I hope this helps!
